File: C#/RefactoringWorkshop/RefactoringWorkshop/Dispensable/DataClasses/Address.cs
namespace RefactoringWorkshop.Dispensable.DataClasses
{
    public class Address
    {
        public string City { get; set; }

        public string PostCode { get; set; }

        public string Country { get; set; }

        public Address(string city, string postCode, string country)
        {
            City = city;
            PostCode = postCode;
            Country = country;
        }
    }
}

File: Java/RefactoringWorkshop/src/main/java/dispensable/dataclasses/Address.java
package dispensable.dataclasses;

public class Address {
    private String city;
    private String postCode;
    private String country;

    public Address(String city, String postCode, String country) {
        this.city = city;
        this.postCode = postCode;
        this.country = country;
    }

    public String getCity() {
        return city;
    }

    public String getPostCode() {
        return postCode;
    }

    public String getCountry() {
        return country;
    }
}
